figo/models.py

```diff
@@ -796,14 +869,19 @@ class Challenge(ModelBase):
         data: challenge data
 
     """
-    __dump_attributes__ = ["id", "title", "label", "format", "data", "type"]
+    __dump_attributes__ = [
+        "id", "title", "label", "format", "data", "type", "location",
+        "created_at"
+    ]
 
     id = None
     title = None
     label = None
     format = None
     data = None
     type = None
+    location = None
+    created_at = None
```

Comment:
This is main fix: 

    __dump_attributes__ = [
        "id", "title", "label", "format", "data", "type", "location",
        "created_at"
    ]

    id = None
    title = None
    label = None
    format = None
    data = None
    type = None
    location = None
    created_at = None

all other staff is clean up code

Reply:
Looks ok! A lot changed because of `black` 👍